forwards_paper.tex

```diff
@@ -966,7 +976,8 @@ \section*{Discussion}
 By using the locations of the mutations on the marginal trees,
 and the fact that these are correlated,
 sequential tree algorithms similar to those in~\citep{kelleher2016efficient}
-can do this in roughly $O(n + m \log n)$ operations.
+can do this in roughly $O(n + m + t)$ operations, where
```

Comment:
My reasoning here: we need O(n + t) time to generate the trees, and a further O(m) to examine each mutation.

Reply:
Each time we examine a mutation, we need to find its allele frequency.  Oh, but this is basically free if pre-computed with tracked_ancestors.  But updating the values in tracked_ancestors takes log(n) steps, so shouldn't this be `n + m + t \log n`?

Reply:
Yes, you're right == O(n + m + t log n)